cmd/utils/flags.go

```diff
@@ -733,6 +733,11 @@ var (
 		Usage: "The multiplier to reduce the SMT depth by when calculating virtual counters",
 		Value: 0.6,
 	}
+	PessimisticProofsCfgFile = cli.StringFlag{
+		Name:  "zkevm.pessimistic-proofs.config",
```

Comment:
Technically, a chain with full execution proof can also use this file, e.g. a full execution proof chain whose initial batch is baked in genesis, and this file will be empty. A name like `zkevm.initial-batch.config` sounds a more generic name that can be used by both pessimistic and full execution proof?